spec/support/origin_info_date_helpers.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
```

Comment:
Since a bunch of these were used both for their class specs and for the new Field class spec, I put them in a spec support file.  Cleans up the specs (I got tired of seeing methods in the files before the specs, which is the practice in this gem).